models/baseline_models.py

```diff
@@ -136,6 +156,52 @@ def predict(self, test_dataset, getdf=False):
             return preds, test_labels
 
 
+class XGBoostModel(BaseModel):
+    def __init__(self, verbose=0):
+        super().__init__()
+        self.verbose=verbose
+
+    def fit(self, train_dataset):
+
+        prior_orders, labels = train_dataset.get_prior_products()
+
+        df_train = get_features(train_dataset, prior_orders, verbose=self.verbose)
+
+        # self.d_train = lgb.Dataset(df_train,
```

Comment:
remove old code if you no longer need it (so maybe at this stage keep it if there's a chance you go back to it, but it shouldn't be in the final version)